localnet/docker-compose.yml

```diff
@@ -243,7 +243,7 @@ services:
       start_period: 10s
 
   deposit-monitor:
-    image: ghcr.io/chainflip-io/chainflip-deposit-monitor/chainflip-deposit-monitor:main-4f97255318399222a92a9f03dfb7381f527b0184
+    image: ghcr.io/chainflip-io/chainflip-deposit-monitor/chainflip-deposit-monitor:version-1-7-85c3192258c31b3a2b93ff609bb31bcca32cf3c9
```

Comment:
@MxmUrw is there a version of the deposit monitor that will work with 1.8 + the recent changes?